internal/vds/vds.cpp

```diff
@@ -22,16 +22,16 @@
 
 using namespace std;
 
-void vdsbuffer_delete(struct vdsbuffer* buf) {
+void response_delete(struct response* buf) {
     if (!buf)
         return;
 
     delete[] buf->data;
     delete[] buf->err;
-    *buf = vdsbuffer {};
+    *buf = response {};
 }
 
-int axis_todim(api_axis_name ax) {
```

Comment:
This doesn't belong in this commit

Reply:
My bad. I tried to fix the commit message with a rewording commit. It seems that I messed something up while squashing the rewording. :(

Reply:
Looks good now, merge when green!